sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -446,15 +447,16 @@ object Radar {
             this.logger = RadarLogger(this.context)
         }
 
-        RadarSettings.updateSessionId(this.context)
-
         if (publishableKey != null) {
             RadarSettings.setPublishableKey(this.context, publishableKey)
         }
 
         if (!this::apiClient.isInitialized) {
             this.apiClient = RadarApiClient(this.context, logger)
         }
+
+        RadarSettings.updateSessionId(this.context)
```

Comment:
now that `updateSessionId()` logs a conversion internally, it has to be called after the apiClient is initialized